enterprise/urls.py

```diff
@@ -15,6 +17,10 @@
         name='grant_data_sharing_permissions'
     ),
     url(
+        r'^enterprise/course/{}/$'.format(settings.COURSE_ID_PATTERN),
```

Comment:
I'm wondering if the URL pattern here should be something more like `r'^enterprise/(?P<enterprise_uuid>[^/]+)/course/{}/enroll/$'.format(settings.COURSE_ID_PATTERN)`? @brittneyexline @mattdrayer @zubair-arbi Thoughts?